Tests/Components Tests/Cash App Pay/CashAppPayComponentTests.swift

```diff
@@ -0,0 +1,265 @@
+//
+//  CashAppPayComponentTests.swift
+//  AdyenUIKitTests
+//
+//  Created by Eren Besel on 4/17/23.
+//  Copyright © 2023 Adyen. All rights reserved.
```

Comment:
This is not our usual copyright header. To avoid this should we make our own .plist file with our usual header and keep in the Xcode shared folder.